### ISSUE TITLE
Magit help command is not useful

### ISSUE THREAD

**Issue by ThorpeJosh** [niyumard/Doom-Emacs-Cheat-Sheet#4]
Hi, Thanks for the summary of commands.

I did find one misleading instruction when following the magit bindings:
"`?` in Magit’s status page for a nice list of available commands and help, `q` to close this help page"

The help pop-up that shows the available commands are not evil bindings. For example under the "Apply changes" section of the help pop-up it has v=Reverse, k=Discard, etc but these are not valid evil bindings.

All the evil bindings can be found below.
https://github.com/emacs-evil/evil-collection/tree/master/modes/magit#key-bindings

Maybe I would just link to the evil-magic bindings page instead of suggesting to use the `?` menu?

**Comment by niyumard** [comment]
Hi!
Thanks for opening an issue.

On my Doom Emacs (which is almost the pure version since my own config file doesn't make a lot of changes to Doom Emacs) `?` is binded to `magit-dispatch` and it indeed shows a list of the right set of bindings:
![ps_20220804135106](https://user-images.githubusercontent.com/42994241/182812132-94f147c7-4625-4c4b-9c4f-fcbfb030a8ff.png)

I believe the issue is caused by your private config files. If you believe the issue is universal please let me know, or otherwise if you find that something in your private config files has caused this, you can share it here. It might be useful for people who will see this issue in the future.

**Comment by ThorpeJosh** [comment]
Thanks for sharing a screenshot of your magit help pop-up!

I came across this issue on the evil-magit repo so I assumed it was a universal issue:
https://github.com/emacs-evil/evil-collection/issues/637

Since you have no issues, I will play around with my config and see if that fixes anything. I have also changed very little in my doom config so I'll investigate and give an update if I find anything.
